libs/utils/src/lib/get-github-count.ts

```diff
@@ -0,0 +1,10 @@
+export const getGithubCount = async () => {
```

Comment:
I wonder if we should establish a convention for server utilities like this? To make it more clear that it cannot be used in a client component. Maybe `fetchGitHubCount`?

Reply:
It could be used in a client in a useEffect.